src/navigator/DrawerNavigator.tsx

```diff
@@ -229,6 +224,10 @@ export default function DrawerNavigator({ route }: Props) {
 
   const shouldShowNftGallery = getFeatureGate(StatsigFeatureGates.SHOW_IN_APP_NFT_GALLERY)
 
+  const shouldShowKeylessBackup = getFeatureGate(
```

Comment:
I think we should abandon the "shouldShow" naming convention everywhere. Even if it's used elsewhere in this same component. The word "should" adds no value.

Reply:
I'll adjust this in #4096, don't want to run CI again here 😄